spec/features/navigation_spec.rb

```diff
@@ -87,7 +87,8 @@
 
   describe "As a merchant" do
     it "I cannot access restricted routes" do
-      user = User.create!(name: "Steve",
+      meg = Merchant.create!(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      user = meg.users.create!(name: "Steve",
```

Comment:
old test failing because we just had a merchant user that is now being redirected to a merchant dashboard but was created with no merchant. added a merchant and we're good to go.